development/compiling/compiling_for_windows.rst

```diff
@@ -169,12 +169,13 @@ dependencies. Running it will bring up the Project Manager.
           :ref:`doc_data_paths_self_contained_mode` by creating a file called
           ``._sc_`` or ``_sc_`` in the ``bin/`` folder.
 
-Development in Visual Studio or other IDEs
-------------------------------------------
+Development in Visual Studio
+----------------------------
 
-For most projects, using only scripting is enough but when development
-in C++ is needed, for creating modules or extending the engine, working
-with an IDE is usually desirable.
+Using an IDE is not required to compile Godot as SCons takes care of everything,
+but if you intend to do engine development or debugging of the C++ code on a
+Windows development environment, you may be interested in configuring things
+for use in Visual Studio.
```

Comment:
```suggestion
Using an IDE is not required to compile Godot, as SCons takes care of everything.
But if you intend to do engine development or debugging of the engine's C++ code,
you may be interested in configuring a code editor or an IDE.

Folder-based editors don't require any particular setup to start working with Godot's
codebase. To edit projects with Visual Studio they need to be set up as a solution.
```

I'd try to make it sound less like VS is the only game in town, and try to explain why we dedicate a whole section to it (it has its own project system in the form of solutions).